### ISSUE TITLE
Is IxPointed the right thing to split out?

### ISSUE THREAD

**Issue by treeowl** [reinerp/indexed#10]
By analogy, perhaps we want `IxApply` instead.

**Comment by typedrat** [comment]
I agree! The only issue is that if I am making breaking changes, I would want to completely readjust the library to follow the `semigroupoids` class structure and probably drastically raise the minimum GHC version by turning `IFunctor` into the quantified constraint it should be.
